.github/workflows/ci.yml

```
@@ -73,3 +73,26 @@ jobs:
       # However, we don't have many projects or targets in this workspace, so we
       # lower the number of agents to reduce spent compute time.
       number-of-agents: 4
+
+  sonarcloud:
+    needs: dte_coordinator
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup
+        uses: ./.github/actions/setup
+      # Uses the cache generated in the distributed step (Needed for the sonar eslint reports).
+      - name: Lint with reports
+        run: yarn nx affected --all --target=lint --configuration=report --parallel=2 --max-warnings=0 --skip-nx-cache
+        # Uses the cache generated in the distributed step (Needed for the sonar jest coverage reports).
+      - name: Tests with coverage
+        run: yarn nx affected --all --target=test --configuration=coverage --parallel=1 --maxWorkers=2 --skip-nx-cache
+
+      # - name: Configure Sonar report paths
+      #   run: yarn configure-sonar-report-paths
+
+      - name: SonarCloud Scan
+        uses: sonarsource/sonarcloud-github-action@master
```

issue: Use versioned tag
```suggestion
        uses: sonarsource/sonarcloud-github-action@v1.7
```